src/main/kotlin/com/exactpro/th2/ws/client/api/impl/WebSocketClient.kt

```diff
@@ -35,7 +34,7 @@ import kotlin.concurrent.withLock
 class WebSocketClient(
     private val uri: URI,
     private val handler: IHandler,
-    private val onMessage: (message: ByteArray, textual: Boolean, direction: Direction) -> Unit,
+    private val onMessage: (message: ByteArray, textual: Boolean, direction: Direction, eventID: EventID?) -> Unit,
```

Comment:
```suggestion
    private val onMessage: (message: ByteArray, textual: Boolean, direction: Direction, eventId: EventID?) -> Unit,
```

Reply:
But isn't EventID class means param gonna be -> eventID by default?

Reply:
Depends on a naming convention you use